tests/core/callbacks/normalization_callback/test_normalization_callback.py

```diff
@@ -0,0 +1,35 @@
+from pytorch_lightning import Trainer, seed_everything
+
+from anomalib.config import get_configurable_parameters
+from anomalib.core.callbacks import get_callbacks
+from anomalib.data import get_datamodule
+from anomalib.models import get_model
+
+
+def run_train_test(config):
+    model = get_model(config)
+    datamodule = get_datamodule(config)
+    callbacks = get_callbacks(config)
+    trainer = Trainer(**config.trainer, callbacks=callbacks)
+    trainer.fit(model=model, datamodule=datamodule)
+    results = trainer.test(model=model, datamodule=datamodule)
+    return results
+
+
+def test_normalizer():
+    config = get_configurable_parameters(model_config_path="anomalib/models/padim/config.yaml")
```

Comment:
Can you add `get_dataset_path()` to `config.dataset.path`? Without this, the CI is downloading the entire dataset again in the current directory.  Due to the permissions of the original files, the CI cannot remove the dataset once the test runs.